specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/cloud-events-schema/incoming_call.json

```
@@ -0,0 +1,38 @@
+{
+  "id": "e80026e7-e298-46ba-bc42-dab0eda92581",
+  "sorce": "/subscriptions/{subscription-id}/resourceGroups/{group-name}/providers/Microsoft.Communication/communicationServices/{communication-services-resource-name}",
```

Should this be `topic`?
```suggestion
  "topic": "/subscriptions/{subscription-id}/resourceGroups/{group-name}/providers/Microsoft.Communication/communicationServices/{communication-services-resource-name}",
```

---

@DominikMe Following the syntax from the other example in the cloud-events-schema folder their topic was renamed to source.

---

Right, cloud event uses source, and you've fixed the typo 👍🏻